### PR TITLE
Run3-alca212 Update the macros and move the fileservice variable from class member to a local variable in Calibration/HcalCalibAlgos

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update the macros and move the fileservice variable from class member to a local variable in Calibration/HcalCalibAlgos

#### PR validation:

Use workflows defined in Calibration/HcalCalibAlgos/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibMonitor.C [hunk 1]
```diff
@@ -1032,16 +1006,19 @@ void CalibMonitor::Loop() {
...
+        if (jp1 >= 0)
+          jp2 = j;
+        if (jp1 < 0)
+          jp1 = j;
       }
     }
-    int jp2 = (jp1 >= 0) ? (int)(ietas_.size() - 1) : jp1;
+    int jp3 = (jp1 >= 0) ? (int)(ietasL_.size() - 1) : jp1;
     if (debug)
       std::cout << "Bin " << kp << ":" << kp1 << ":" << kv << ":" << kv1 << ":" << kd << ":" << kd1 << ":" << jp << ":"
```
[the remark this answers is not in the document]
Comment (perrotta) [reply Calibration/HcalCalibAlgos/macros/CalibMonitor.C:1019]: @tvami this is a macro to be run interactively in root (if I understand it correctly); and in any case the `cout` is shielded by a `debug` flag set `false` by default: I wouldn't push for including LogMessages for this user debug options.

[the remark this answers is not in the document]
Comment (tvami) [reply Calibration/HcalCalibAlgos/macros/CalibMonitor.C:1019]: ok! sounds good!

**Comment by tvami** [comment]
+alca
  * tests pass, diffs in MsgLogger only
  * code changes in line w PR describtion

**Comment by qliphy** [comment]
+1

**Pull request merged**